### PR TITLE
Move files_drop to compiled handlebars

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
LGTM

File: apps/files_sharing/js/templates/files_drop.handlebars [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<li data-toggle="tooltip" title="{{name}}" data-name="{{name}}">
+	'{{#if isUploading}}
+		<span class="icon-loading-small"></span> {{name}}
+	{{else}}
+		<img src="{{iconSrc}}"/> {{name}}
```
Comment (danielkesselberg) [inline apps/files_sharing/js/templates/files_drop.handlebars]: I would prefer `` over ``  :smiling_imp:

Comment (ChristophWurst) [reply apps/files_sharing/js/templates/files_drop.handlebars]: It was like this in the original template: https://github.com/nextcloud/server/pull/11839/files#diff-fb8a2ae938c88d82cafb8b46f5e7b400L17

IMO we should rather change this in a follow-up PR than this one.

**Comment by MorrisJobke** [comment]
Conflicts 🙈

**Comment by MorrisJobke** [comment]
> Fixed and rebased

Quite some failures. The JSUnit one I I guess one that is really a problem.

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Pull request merged**